### PR TITLE
pd: add new configurations

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### First-time contributors' checklist
- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
We have added a new configuration `region-score-formula-version` and updated the default value of `enable-cross-table-merge` for v5.0.

### Which TiDB version(s) do your changes apply to? (Required)
- [x] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?
- This PR is translated from:
- Other reference link(s): https://github.com/tikv/pd/pull/3269 https://github.com/tikv/pd/pull/3129

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lhy1024 — APPROVED** [review]

File: pd-control.md [hunk 1]
```diff
@@ -304,6 +305,16 @@ export PD_ADDR=http://127.0.0.1:2379 &&
     >> config set key-type raw
     ```
 
+- `region-score-formula-version` 用于设置 Region 算分公式的版本，支持的值有 `["v1", "v2"]`。v2 版本公式有助于减少上下线等场景下冗余的 balance region 调度。
```
Comment (TomShawn) [inline pd-control.md:308]:
```suggestion
- `region-score-formula-version` 用于设置 Region 算分公式的版本，支持的值有 `["v1", "v2"]`。v2 版本公式有助于减少上下线等场景下冗余的 balance Region 调度。
```

File: pd-control.md [hunk 2]
```diff
@@ -398,7 +409,7 @@ Leader 调度的开销较小，需要的时候可以适当调大。
     config set low-space-ratio 0.9
     ```
 
-- `high-space-ratio` 用于设置 store 空间充裕的阈值。当节点的空间占用比例小于指定值时，PD 调度时会忽略剩余空间这个指标，主要针对实际数据量进行均衡。
+- `high-space-ratio` 用于设置 store 空间充裕的阈值，此配置仅的 `region-score-formula-version = v1` 时生效。当节点的空间占用比例小于指定值时，PD 调度时会忽略剩余空间这个指标，主要针对实际数据量进行均衡。
```
Comment (TomShawn) [inline pd-control.md:412]:
```suggestion
- `high-space-ratio` 用于设置 store 空间充裕的阈值，此配置仅的在 `region-score-formula-version = v1` 时生效。当节点的空间占用比例小于指定值时，PD 调度时会忽略剩余空间这个指标，主要针对实际数据量进行均衡。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
Follow-up: 添加引入版本的标记

**Pull request merged**